packages/lab/src/option/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./Option";
+export * from "./OptionGroup";
```

Comment:
do we want to export OptionList too?

Reply:
At the moment, I don't think it's necessary. It's pretty much an internal component for now.